### DISCUSSION TITLE
Procedurally generated audio

### DISCUSSION THREAD

**Discussion by guillaumebrunerie** [remotion-dev/remotion#3177] [Q&A]
Hello,

I would like to make some procedurally generated videos, and Remotion seems to be a great tool for that! But I would also like to have procedurally generated audio, but there doesn’t seem to be much support for it in Remotion, the only thing I could find is the ability to play certain sound files at certains points in time, which is not very flexible and probably doesn’t scale well either.

Is there a better way to do procedurally generated audio? For instance a way to export WebAudio to a file that we can then import in Remotion, or a similar project to Remotion but for audio instead?

**Comment by JonnyBurger** [comment]
I have little expertise in this to build it out further. We'll focus on video for now, but.. some simple things are possible!

Check out this Tone.JS example: https://github.com/remotion-dev/tone-js-example
